recipes/kernax/meta.yaml

```diff
@@ -0,0 +1,53 @@
+package:
+  name: kernax
+  version: 0.0.1
```

Comment:
The `version` here should match the package version (`0.1.9`) and then you can use the jinja tags in the `url`